src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -260,6 +260,57 @@ public TypeSymbolWithAnnotations SetIsAnnotated(CSharpCompilation compilation)
         public TypeSymbolWithAnnotations AsNullableReferenceType() => _extensions.AsNullableReferenceType(this);
         public TypeSymbolWithAnnotations AsNotNullableReferenceType() => _extensions.AsNotNullableReferenceType(this);
 
+        public bool TryMergeNullability(TypeSymbolWithAnnotations other, VarianceKind variance, out TypeSymbolWithAnnotations merged)
+        {
+            TypeSymbol type;
+            bool? isNullable;
+            if (TypeSymbol.TryMergeNullability(other.TypeSymbol, out type) &&
```

Comment:
>TryMergeNullability [](start = 27, length = 19)

This appears to drop variance. The variance from the enclosing levels are required. Below invariant, all merges must be invariant. Below an `in` type parameters, the role of covariant and contravariant are inverted.